pkg/skaffold/kubernetes/manifest/visitor.go

```diff
@@ -123,23 +124,27 @@ type recursiveVisitorDecorator struct {
 	delegate FieldVisitor
 }
 
-func (d *recursiveVisitorDecorator) Visit(o map[string]interface{}, k string, v interface{}) bool {
-	if d.delegate.Visit(o, k, v) {
-		visitFields(v, d)
+func (d *recursiveVisitorDecorator) Visit(path string, o map[string]interface{}, k string, v interface{}) bool {
+	if d.delegate.Visit(path, o, k, v) {
+		visitFields(path, v, d)
 	}
 	return false
 }
 
 // visitFields traverses all fields and calls the visitor for each.
-func visitFields(o interface{}, visitor FieldVisitor) {
+func visitFields(jsonpath string, o interface{}, visitor FieldVisitor) {
 	switch entries := o.(type) {
 	case []interface{}:
 		for _, v := range entries {
-			visitFields(v, visitor)
+			visitFields(jsonpath, v, visitor)
```

Comment:
You need to update `jsonpath` here too, right?

Reply:
No, this case comes up when a list or map is encountered in the json.  The []interface{} here is always a map of maps and when we actually traverse those the other case is called.  Additionally what would you append to jsonpath in this case, `fmt.Sprintf("%v", v)`?  The other case hits every field in the json allowing us to have jsonpaths to every field.

Reply:
Could you add a comment to this effect?  Even if `// this case covers lists so we don't update the path`

Reply:
Good idea, done